Makefile

```diff
@@ -36,10 +36,17 @@ TESTFLAGS    :=
 DUPLFLAGS    := -t 100
 
 ifeq ($(STATIC),1)
+# Static linking with glibc is a bad time. If we're building statically,
+# assume musl is installed and use that.
```

Comment:
This seems like a big assumption (it's linux specific for one thing). I'd rather remove this section than leave in a little-tested and unused configuration that we're not even sure anyone wants.